Makefile

```diff
@@ -1,14 +1,21 @@
 BUILD_DIR=build
 EXAMPLES=api-gateway kinesis s3
 
-.PHONY: default
```

Comment:
Why remove the `.PHONY` targets?

Reply:
I found them noisy and they're only necessary if the target name is the same as file or directory name. 

I can put them back, if prefer that.